src/ok.rs

```diff
@@ -170,7 +174,7 @@ pub fn cmd(gen: Option<SevGeneration>, quiet: bool) -> Result<()> {
         }
 
         let write_test = &mut sysfs_tests[1](fs::OpenOptions::new().write(true));
-        write_test.msg = format!("Writing to /dev/sev: {}", write_test.msg);
+        write_test.msg = format!("Writing to /dev/sev: {}\n", write_test.msg);
```

Comment:
No newlines in a singly formatted string. It is the caller's responsibility to format how data is emitted, _not_ the test.